docs/source/basic_tutorial.rst

```diff
@@ -0,0 +1,216 @@
+..
+  Copyright (c) 2017-present, Facebook, Inc.
+  All rights reserved.
+  This source code is licensed under the BSD-style license found in the
+  LICENSE file in the root directory of this source tree. An additional grant
+  of patent rights can be found in the PATENTS file in the same directory.
+
+What is ParlAI?
+===============
+
+It's a python-based platform for enabling dialog AI research.
+
+Its goal is to provide researchers:
+
+- a unified framework for training and testing dialog models
+- multi-task training over many datasets at once
+- seamless integration of Amazon Mechanical Turk for data collection and human evaluation
+
+
+Getting Started
+---------------
+
+Agents
+^^^^^^
+
+The most basic concept in ParlAI is an Agent.
+Agents can be humans, a simple bot which repeats back anything that it hears,
+your perfectly-tuned neural network, a dataset being read out,
+or anything else that might send messages or interact with its environment.
+
+Agents have two primary methods they need to define:
+
+.. code-block:: python
+
+    def observe(self, observation)
+    def act(self)
+
+
+``observe`` notifies the agent of an action taken by another agent.
+
+``act`` produces an action from the agent.
+
+Observations are structured as a python `dict` with the following fields:
+
+
+.. image:: _static/img/act-obs-dict.png
+    :width: 60 %
+
+
+All of these fields are technically optional, and each task is free to use them
```

Comment:
"as appropriate" -- well within limits.  its good if they are similar to other tasks as possible somehow? i would maybe reword a little..